tests/page-objects/default/enketo/generic-form.wdio.page.js

```diff
@@ -65,6 +77,7 @@ const verifyReport = async () => {
 };
 
 const submitForm = async () => {
+  await browser.waitUntil(async () => (await $$('.invalid-required')).length === 0);
```

Comment:
This code is repeated 3 times, also on line 16 and 19. Pull out a `hasValiationErrors` function?